assets/css/style.css

```diff
@@ -0,0 +1,23 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="Stylesheet" href="./assets/css/style.css">
+    <title>Mystery Melody</title>
+</head>
+
+<style>

```

Comment:
Yea, like I said earlier. Back in the first two weeks of bootcamp we learned how to style the page only on the css file, so that you don't have to write it directly on the html. Using the <style> tag in html is called "inline styling", which Mohammed said back in the beginning of class that it's frowned upon. 
Granted, what you're doing works, and can easily be copy-pasted onto the css file.